src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/BindingStep/help.html

```diff
@@ -73,13 +73,15 @@
 foo'bar
 </code></pre>
 <p>
-    Mangled secrets are not detected by Jenkins and will appear in plain text in the log file, in this case as:
+    Mangled secrets can only be detected on a best-effort basis. By default, Jenkins will attempt to mask mangled
+    secrets as they would appear in output of Bash and Windows batch. Without these strategies in place, mangled secrets
+    would appear in plain text in log files. In the example above, this would result in:
```

Comment:
BTW for natural-language text files it is a good idea to use one physical line per sentence or major phrase. Easier to review & edit.